security/csp-report/index.tpml

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
Let's rename this file to `index-template.html`. This way we will get syntax highlighting and it will be understandable that this file is a template.